resource-directory/test/service.go

```diff
@@ -30,7 +30,7 @@ func MakeConfig(t *testing.T) service.Config {
 	cfg.Clients.Eventstore.ProjectionCacheExpiration = time.Second * 120
 
 	cfg.ExposedHubConfiguration.CAPool = config.CA_POOL
-	cfg.ExposedHubConfiguration.AuthorizationServer = "https://" + config.OAUTH_SERVER_HOST
+	cfg.ExposedHubConfiguration.Authority = "https://" + config.OAUTH_SERVER_HOST
```

Comment:
Perfect that it runs in tests but it's not working in real. Bundle, hub nor helm charts.